dot/network/host_test.go

```diff
@@ -548,14 +556,15 @@ func TestStreamCloseEOF(t *testing.T) {
 
 	stream, err := nodeA.host.send(addrInfoB.ID, nodeB.host.protocolID, testBlockReqMessage)
 	require.NoError(t, err)
-	require.False(t, handler.exit)
 
 	err = stream.Close()
 	require.NoError(t, err)
 
-	time.Sleep(TestBackoffTimeout)
-
-	require.True(t, handler.exit)
+	select {
+	case <-time.After(time.Millisecond * 500):
```

Comment:
nit not a big deal, but using a `timer.NewTimer(duration)` is better since `time.After` will leak and have its channel blocked forever if `case <-handler.exitChan:` triggers before it, since you cannot stop the underlying timer. Again this is a test so doesn't matter, but I felt like sHaRiNg 😉